test/run-tests.js

```diff
@@ -1,7 +1,7 @@
 const npmRun = require('npm-run')
 const fs = require('fs')
 const path = require('path')
-const LATEST_VERSION = '16.6'
+const LATEST_VERSION = '16.8'
```

Comment:
I included this in the PR since this version mismatches with the one in the `install-test-deps.js`-script on master, creating problems when running tests.